Sources/ConsoleKitTerminal/Docs.docc/index.md

```diff
@@ -8,4 +8,4 @@
 
 * Utilities for sending text (including styles and colors, when supported) to a terminal.
 * Utilities for reading input from a terminal.
-* ``ConsoleLog`` and ``ConsoleFragmentLogger``, [SwiftLog](https://github.com/apple/swift-log) `LogHandler` implementations for customizable logging to a console.
+* ``ConsoleLogger`` and ``ConsoleFragmentLogger``, [SwiftLog](https://github.com/apple/swift-log) ``LogHandler`` implementations for customizable logging to a console.
```

Comment:
```suggestion
* ``ConsoleLogger`` and ``ConsoleFragmentLogger``, [SwiftLog](https://github.com/apple/swift-log) `LogHandler` implementations for customizable logging to a console.
```
The lack of a symbol link on `LogHandler` is deliberate - DocC still doesn't support cross-module linking, so the link won't work anyway.

Reply:
is this causing DocC errors? other rendering engines (like Swiftinit) have no problem resolving cross-package links

Reply:
TBH I don't recall offhand whether it causes an error or just a warning.